### DISCUSSION TITLE
SEO for image alt

### DISCUSSION THREAD

**Discussion by ideateXYZ** [Sped0n/bridget#387] [Q&A] [answered]
Hi! I really really love the theme but I couldnt figure out how to properly do alt for images for SEO purposes - how would I optimise for SEO with bridget theme? Thank you!

**Comment by 713b76b7-2247-4bd5-98a7-014226363f9b** [comment]
SEO is outside the scope of my knowledge o_o

**Comment by AbdulQadir034** [comment, marked as answer]
Hello @dittoworks performing SEO for image Alt text is quite easy. you have to define the image and add a your focus keyword/phrase in that description of image. for example: my image is of boxing gloves, so i will explain whats in that image and add my focus keyword like [boxing gloves](https://www.wickedboxing.com/search?q=Boxing+Gloves) in it. thats it!

**Comment by Farhan7585** [comment]
Hi there!
I would suggest you install a plugin named Rank Math Seo. It will work for you. If not then When you will upload any image you will see 4 boxes before uploading your image.
1. Alt text for image
in this You can add image text I mean your keyword like if you are uploading image with tile High calorie breakfast then you add high calorie breakfast in Alt text box
2. Title
in this you need to add title of article.
3. Caption 
in this caption box you add the name of website or source from where you take picture
4. Description
in this box you will add description of image. for which purpose you are adding image.
